### PR TITLE
ASoC: intel: sof_rt5682: set platform name for dai links

### PR DESCRIPTION
**Pull request by ranj063**
Set platform name for dai links.

Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>

### CODE REVIEW

File: sound/soc/intel/boards/sof_rt5682.c [hunk 1]
```diff
@@ -346,8 +347,9 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 static int sof_audio_probe(struct platform_device *pdev)
 {
 	struct snd_soc_dai_link *dai_links;
+	struct snd_soc_acpi_mach *mach = (&pdev->dev)->platform_data;
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_rt5682.c]: try to follow the other machine drivers and put this in the code block, not in declarations.

File: sound/soc/intel/boards/sof_rt5682.c [hunk 2]
```diff
@@ -346,8 +347,9 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
...
 	struct sof_card_private *ctx;
-	int dmic_num, hdmi_num;
+	int dmic_num, hdmi_num, ret;
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_rt5682.c]: separate line for ret

File: sound/soc/intel/boards/sof_rt5682.c [hunk 3]
```diff
@@ -189,7 +190,7 @@ static struct snd_soc_ops sof_rt5682_ops = {
 
 static struct snd_soc_dai_link_component platform_component[] = {
 	{
-		.name = "sof-audio"
+		.name = "0000:00:1f.3"
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_rt5682.c:194]: add comment that this might overridden.

**Comment by ranj063** [comment]
@plbossart updated now

**Pull request merged**